schema/types.go

```diff
@@ -272,6 +272,23 @@ func (c CQTypes) Len() int {
 	return len(c)
 }
 
+func (c CQTypes) String() string {
+	var sb strings.Builder
+	sb.WriteString("[")
+	for i, v := range c {
+		if i > 0 {
+			sb.WriteString(",")
+		}
+		if v == nil {
+			sb.WriteString("nil")
+		} else {
+			sb.WriteString(v.String())
+		}
+	}
+	sb.WriteString("]")
+	return sb.String()
+}
```

Comment:
I don't think we should be adding this public function because it is so specific for the one output we are using for reporting the PK dups

Reply:
Initially it was there, did something change for v2?

Reply:
I didn't add it, it was already in place below, however, I moved it a bit to top, as there's a mark for functions used only in tests (and now it isn't used in tests only).